ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```
@@ -1,11 +1,12 @@
 # Defaults file for the compatibility layer role.
 ---
-eessi_version: "2023.06"
+eessi_version: "2025.01"
```

@bedroge Let's bump this to `2025.02`?

```suggestion
eessi_version: "2025.02"
```

---

Yes, definitely, though it also requires the set to be renamed/merged. I'll do that first.

---

Done in https://github.com/EESSI/compatibility-layer/pull/209/commits/0b67029beda4bbcc55119dc2140217e2e6e1a668.